css/properties/image-orientation.json

```diff
@@ -53,7 +53,7 @@
         },
         "from-image": {
```

Comment:
This is not about from-image then anymore, so the identifier should probably be changed to maybe "flip_and_angle".

Reply:
good catch. thanks.